modin/pandas/indexing.py

```diff
@@ -813,6 +813,17 @@ def _set_item_existing_loc(self, row_loc, col_loc, item):
         item : modin.pandas.DataFrame, modin.pandas.Series or scalar
             Value that should be assigned to located dataset.
         """
+        if (
```

Comment:
@dchigarev, should we add a condition to not get into this case for HDK because it would be defaulting to pandas? How does HDK handle the case so far?

Reply:
Well, HDK currently defaults to pandas anyway on setting a scalar item. Before your changes, the further execution would end up either in `qc.write_items` or in `qc.setitem` and neither of these functions are supporting a scalar case.

Reply:
Ok, thanks